code/_onclick/item_attack.dm

```diff
@@ -3,6 +3,9 @@
 /obj/item/proc/attack_self(mob/user)
 	return
 
+/obj/item/proc/pre_attackby(obj/O, mob/living/user, params) //do stuff before attackby!
+	return TRUE //return FALSE to avoid calling attackby after this proc does stuff

```

Comment:
uh yeah? returning FALSE would prevent any attackby from working, as the comment says right here